exercises/risk-risiko.cpp

```diff
@@ -27,3 +26,48 @@
   M 3 vs 3 => blue win
   O 2 vs 1 => red win
 */
+#include <iostream>
+#include <ctime>
+using namespace std;
+
+void swap(int& a, int& b) {
+    int temp = a;
+    a = b;
+    b = temp;
+}
+
+void selectionsort(int array[], int n) {
+    int indexMax;
+    for (int i = 0; i < n; i++) {
+        indexMax = i;
+        for (int j = i + 1; j < n; j++) {
+            if (array[j] > array[indexMax])
+                indexMax = j;
+        }
+        swap(array[i], array[indexMax]);
+    }
+}
+
+int main() {
+    srand(time(0));
+    int Red[3] = { rand() % 7,rand() % 7 ,rand() % 7 };
+    int Blue[3] = { rand() % 7,rand() % 7 ,rand() % 7 };
```

Comment:
let's wrap this into a separate function:
```C++
int[] rollDices(int n = 3) {
    int[n] dices;
    for (int i = 0; i < n; i++) {
        dices[i] = rand() % 7;
    }
    
    return dices;
}

int main() {
  int Red[3] = rollDices();
}
```

I did not test the code, but the idea is to reduce the redundant code and make this initialization re-usable